src/ui/theme/colors.ts

```diff
@@ -51,4 +51,8 @@ export const COLORS = {
     800: '#0F0FA9',
     900: '#0B0A63',
   },
+  others: {
+    lilac: '#9985FF',
+    clarity: '#C1D21B',
```

Comment:
Does this refer to Clarity language? I recommend using a universal color name, e.g. `limeGreen` might be a suitable alternative.

Reply:
I was just following the design spec here https://www.figma.com/file/6JPmaDsHZ2fHfOyNY7sIZY/Stacks-Explorer-UI-Theme?type=design&node-id=1-203&mode=design

Reply:
@eugeniadigon is `Clarity` a color name? or should we use something like `limeGreen`?

Reply:
Yes, because it represents [Clarity's](https://clarity-lang.org/) brand color, but open to change it to `lime` if that makes it easier